pages/page.py

```diff
@@ -115,15 +97,9 @@ def return_to_previous_page(self):
     def get_relative_path(self, url):
         self.selenium.get(u'%s%s' % (self.base_url, url))
 
-    def find_element(self, *locator):
```

Comment:
I know this was merged but just to understand

Why did we removed this methods?